config/opal_check_singularity.m4

```diff
@@ -0,0 +1,52 @@
+# -*- shell-script ; indent-tabs-mode:nil -*-
+#
+# Copyright (c) 2016      Intel, Inc. All rights reserved.
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+# OPAL_CHECK_SINGULARITY(prefix, [action-if-found], [action-if-not-found])
+# --------------------------------------------------------
+AC_DEFUN([OPAL_CHECK_SINGULARITY],[
+    OPAL_VAR_SCOPE_PUSH([spath have_singularity])
+
+    AC_ARG_WITH([singularity],
+                [AC_HELP_STRING([--with-singularity(=DIR)],
+                                [Build support for the Singularity container, optionally adding DIR to the search path (default: no)])],
+                                [], with_singularity=no)
```

Comment:
It took a little time to puzzle out this statement -- isn't the `[], with_singularity=no)` line mis-indented?  I.e., it's not part of `AC_HELP_STRING` -- they're arguments to `AC_ARG_WITH`.

I don't usually comment on indenting, but please fix this, because m4 is hard enough to read.  :smile: